src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -271,6 +612,25 @@ public void Dispose()
                 return lines;
             }
 
+            public async Task<string> ReadRequestContentAsync()
```

Comment:
I'm not following this method.  It looks like you find the Content-Length, skip the next line, and then read whatever is after this until you get a new line.  Isn't this going to produce weird results in many cases?

Reply:
I assumed the stuff after content-length was content, seems like that is wrong. Will revert this.